WebdriverIO.md

```diff
@@ -182,10 +185,27 @@ var openTopic = async function(topicName) {
 }
 ```
 
+As you may have noted above that `$$` selector is used by calling a function instead of getting directly declared as variable like `$`, the reason is we cannot use `$$` without `await` prefix as this leads to flakiness in test because it returns an unresolved promise, and we can use `await` only inside async function. So as we need to place the selectors at top we are creating a function instead and using `await` prefix wherever we are calling the function.
+
```

Comment:
>and we can use `await` only inside async function

I am not able to understand this. I think we use await prefix inside async function only, right? For eg see openTopic is an async function only. Hence we are able to use await.

Reply:
My comment was why we can't use `await $$selector` at the top instead of having a function for it and then calling it because theoritically both would mean the same. This is in reference to what you said above i.e we need an await for $$ selector.

Reply:
Is it fine now?

Reply:
PTAL at https://github.com/oppia/oppia-web-developer-docs/pull/11#discussion_r922803180